build/package/windows/nri-amd64-installer/Product.wxs

```diff
@@ -67,29 +67,32 @@
 
     <Fragment>
         <ComponentGroup Id="CustomPluginsBinComponent" Directory="CustomPluginsBinFolder">
-            <Component Id="CMP_NRI_{IntegrationName}_EXE" Guid="eea30c6e-7a14-4cc5-97c5-88a742041643" Win64="yes">
-                <File Id="FILE_NRI_{IntegrationName}_EXE"
-                      Source="$(var.BinariesPath){IntegrationExe}"
+            <Component Id="CMP_NRI_$(var.IntegrationName)_EXE" Guid="eea30c6e-7a14-4cc5-97c5-88a742041643" Win64="yes">

```

Comment:
I am not sure why we are changing all Guid, it should not be an issue but I am not 100% sure

Reply:
I don't remember making this change so I guess this was done before we took over. I'm not that knowledgeable of windows "package management" so I don't really know the implications of this. Maybe someone can clarify whether this is desired or needed somewhere else? @brushknight @ardias @gsanchezgavier

Reply:
also @davidbrota

Reply:
We decided to keep them as they were in `master` reverting previous changes if necessary.